### PR TITLE
docs(flutter_weather): add web support

### PR DESCRIPTION
**Pull request by dawidd6**
## Status

**READY**

## Breaking Changes

NO

## Description

Adds web support to flutter_weather example.

Closes: #2361

## Type of Change

- [ ] ✨ New feature (non-breaking change which adds functionality)
- [ ] 🛠️ Bug fix (non-breaking change which fixes an issue)
- [ ] ❌ Breaking change (fix or feature that would cause existing functionality to change)
- [ ] 🧹 Code refactor
- [ ] ✅ Build configuration change
- [x] 📝 Documentation
- [ ] 🗑️ Chore

### CODE REVIEW

**Comment by Gene-Dana** [comment]
Hi there ! 

I was researching why web support was not working for this, and I think I narrowed it down to an issue with Hydrated Bloc.

Perhaps this a potential solution?
image

File: examples/flutter_weather/web/index.html [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+<html>
+<head>
+  <!--
+    If you are serving your web app in a path other than the root, change the
+    href value below to reflect the base path you are serving from.
+
+    The path provided below has to start and end with a slash "/" in order for
+    it to work correctly.
+
+    Fore more details:
```
Comment (felangel) [inline examples/flutter_weather/web/index.html:11]:
```suggestion
    For more details:
```

**Comment by felangel** [comment]
Unfortunately it looks like metaweather does not support CORS which means we'll need to migrate to a different weather api. I think we can still merge this for now and address the migration in a separate PR 👍

**Review by felangel — APPROVED** [review]
LGTM ✅ 
Thanks again for the contribution! 💯 🚀

**Pull request merged**